enhancements/declarative-index-config.md

```diff
@@ -788,10 +794,10 @@ The `properties.<"olm.skipRange">` property is currently consumed only by the OL
 
 #### Creating the json/yaml config file to represent a package
 
-The `opm index add` command will be enhanced to create a config file for representing the package the bundle is being added to if it is the first bundle in the package that is being added to this index. If the package the bundle is being added to already exists, the `olm.bundle` blob for the bundle will be added to the existing package config file.
+A new command `opm alpha add` command will create a config file for representing the package the bundle is being added to if it is the first bundle in the package that is being added to this index. If the package the bundle is being added to already exists, the `olm.bundle` blob for the bundle will be added to the existing package config file.
 
 ```bash
-$ opm index inspect --image=docker.io/my-namespace/community-operators:latest --output=json
+$ opm unpack docker.io/my-namespace/community-operators:latest 
```

Comment:
I think we should consolidate this with https://hackmd.io/74wnS33ISkGu_v0Qfh_tZQ?view before merging.

cc @joelanford @anik120 @ecordell

Reply:
Replied here https://coreos.slack.com/archives/G3T7N42NP/p1618848760060500?thread_ts=1618840897.055900&cid=G3T7N42NP